src/Components/JSONDetails.tsx

```diff
@@ -19,7 +20,43 @@ export const JSONDetails: React.FC<IProps> = props => {
   const theme = useContext(ThemeContext);
   const [viewJSON, setViewJSON] = useState<boolean>(false);
 
-  const _renderLabel = ([raw]: string) => <Text style={{ fontSize: 14, color: theme.base0D }}>{`${raw} : `}</Text>;
+  const _renderLabel = viewJSON
+    ? ([raw]: string) => <Text style={{ fontSize: 14, color: theme.base0D }}>{`${raw} : `}</Text>
+    : '';
+  let _content = null;
+  try {
+    _content = (
+      <View style={[styles.container, { position: 'relative' }]}>
+        <ScrollView horizontal style={{ backgroundColor: theme.secondaryColor }}>
+          <ScrollView nestedScrollEnabled style={styles.scrollview}>
+            {viewJSON ? (
+              <JSONTree
+                data={typeof props.data === 'string' ? JSON.parse(props.data) : props.data}
+                labelRenderer={_renderLabel}
+                theme={theme}
+                invertTheme={false}
+              />
+            ) : (
+              <Text style={{ color: theme.textColorOne }}>
+                {typeof props.data !== 'string' ? JSON.stringify(props.data, null, 2) : props.data}
+              </Text>
+            )}
+          </ScrollView>
+        </ScrollView>
+      </View>
+    );
+  } catch (error) {
+    _content = (
+      <View style={[styles.container, { position: 'relative', backgroundColor: theme.secondaryColor, padding: 16 }]}>
+        <Title style={{ color: theme.textColorOne, marginBottom: 16 }}>
```

Comment:
why we don't have a scrollview here too?

Reply:
It's just an error page to indicate that something goes wrong. If the user want to see the JSON stringified entirely, I can switch to plain text mode. But I can add a vertical scrollview here too.